SeleniumPracticalExercise/PageObjects/VerifyEmployee.cs

```diff
@@ -0,0 +1,57 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.DevTools.V125.CSS;
+using OpenQA.Selenium.DevTools.V125.Page;
+using SeleniumPracticalExercise.PageObjects.Common;
+
+
+namespace SeleniumPracticalExercise.PageObjects
+{
+    public class VerifyEmployee : BasePageLocal
+    {
+        private readonly By EmployeeId = By.CssSelector("div.oxd-grid-2 input");     
+        private readonly By SaveButton = By.CssSelector("button[type='submit']");   
+        private readonly By PIMButton = By.CssSelector("a[href='/web/index.php/pim/viewPimModule']");   
+        private readonly By TxtBoxEmployeeID = By.CssSelector("div.oxd-input-group input.oxd-input");  
+        private readonly By SubmitBtn = By.CssSelector("button[type='submit']");  
+        private readonly By ExpectedID = By.XPath("//div[contains(@class,'oxd-table-row--clickable')]//div[text()=''][2]/div");  
+        private readonly By ExpectedFirstName = By.XPath("//div[contains(@class,'oxd-table-row--clickable')]//div[text()=''][3]/div");  
+        private readonly By ExpectedLastName = By.XPath("//div[contains(@class,'oxd-table-row--clickable')]//div[text()=''][4]/div");   
+        public VerifyEmployee(IWebDriver driver) : base(driver)
+        {
+
+        }
+        public void ClickSave()
+        {
+            Click(SaveButton);
+        }
+        public void ClickPIM()
+        {
+            Click(PIMButton);
+        }
+        public string IDNum() // get Emp ID value 
```

Comment:
In general, if you are getting a value from the page, name your method, `GetX()`, e.g. `GetId()`. If you are setting a value, name your method, `SetX()`, e.g. `SetId()`.

Reply:
confused how to do this. I tried this below and I'm getting an error message. 
       public string IDNum()
        {
            get 
            { 
                return EmployeeId; 
            }
            set 
            { 
                EmployeeId = Value; 
            }
            
        }

Reply:
The syntax you are using for "getters" and "setters". That applies to fields on a class. I'm just talking about method naming... any method that gets a value should be named, `GetSomething()` and any method that sets a value should be named, `SetSomething()`.